public/docs/ts/latest/guide/architecture.jade

```diff
@@ -1,190 +1,218 @@
-include ../_util-fns
+block includes
+  include ../_util-fns
+  - var _library_module = 'library module'
+  - var _at_angular = '@angular'
 
 :marked
-  Angular 2 is a framework to help us build client applications in HTML and JavaScript.
-  
-  The framework consists of several cooperating libraries, some of them core and some optional.
-  
-  We write applications by composing HTML *templates* with Angularized-markup, 
-  writing *component* classes to manage those templates, adding application logic in *services*,
-  and handing the top root component to Angular's *bootstrapper*. 
-  
-  Angular takes over, presenting our application content in a browser and responding to user interactions
-  according to the instructions we provided.
-  
-<!-- figure img(src="/resources/images/devguide/architecture/airplane.png" alt="Us" align="left" style="width:200px; margin-left:-40px;margin-right:10px" ) -->
+  Angular 2 is a framework to help us build client applications in HTML and
+  either JavaScript or a language (like Dart or TypeScript) that compiles to JavaScript.
+
+block angular-parts
+  :marked
+    The framework consists of several cooperating libraries, some of them core and some optional.
+
 :marked
-  Of course there is more to it than this. We'll learn the details when we dive into the guide chapters.
+  With Angular, we write applications by composing HTML *templates* with Angularized-markup,
+  writing *component* classes to manage those templates, adding application logic in *services*,
+  and handing the top root component to Angular's *bootstrapper*.
+
+  Angular takes over, presenting our application content in a browser and
+  responding to user interactions according to the instructions we provided.
+
+  Of course there is more to it than this.
+  We'll learn the details when we dive into the guide chapters.
   Let's get the big picture first.
 
 figure
   img(src="/resources/images/devguide/architecture/overview2.png" alt="overview" style="margin-left:-40px;" width="700")
+
 :marked
   The architecture diagram identifies the eight main building blocks of an Angular 2 application:
-  1. [Module](#module)   
-  1. [Component](#component)
-  1. [Template](#template)
+
+  1. [Modules](#modules)
```

Comment:
I had left the Modules section out of the Dart version because (1) it was added late to the TS version (which is why we already had a handy, module-less image) and (2) it seemed like it wasn't really necessary, from a Dart perspective.

Are "modules" a concept used in other pages?